resident/resident-service/src/main/java/io/mosip/resident/util/Utility.java

```diff
@@ -275,16 +276,23 @@ public Map<String, Object> getMailingAttributes(String id, Set<String> templateL
 		return attributes;
 	}
 
-	private String getPreferredLanguage(JSONObject demographicIdentity) {
+	private Set<String> getPreferredLanguage(JSONObject demographicIdentity) {
 		String preferredLang = null;
 		String preferredLangAttribute = env.getProperty("mosip.default.user-preferred-language-attribute");
 		if (!StringUtils.isBlank(preferredLangAttribute)) {
 			Object object = demographicIdentity.get(preferredLangAttribute);
 			if(object!=null) {
 				preferredLang = String.valueOf(object);
+				if(preferredLang.contains(ResidentConstants.COMMA)){
+					String[] preferredLangArray = preferredLang.split(ResidentConstants.COMMA);
+					return new HashSet<>(Arrays.asList(preferredLangArray));

```

Comment:
I think you can use Arrays.asSet method. Please check.

Reply:
There are no such method Arrays.asSet
simplified above
`return Arrays.stream(preferredLangArray).collect(Collectors.toSet());`